### PR TITLE
some sporing improvements

### PR DESCRIPTION
**Pull request by Tjellez**
- [ ] [API update message](https://github.com/bring/developer-site#tips-for-writing-an-api-update-message) (changelog entry) has been reviewed by Mybring Experience.

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/zjAbaAge](https://trello.com/c/zjAbaAge)

File: content/api/tracking/index.md [hunk 1]
```diff
@@ -46,14 +46,14 @@ important:
      With reference to the change, the existing field "dateOfEstimatedDelivery" will have the correct information regarding delivery date and therefore, the field "dateOfDelivery" in the open as well as logged in Tracking API versions will be set to Blank from 01.Jan.2021 for backward compatibility and gradually be removed effective from date 01.Mar.2021.
 
 introduction: |
-  The Tracking API provides the opportunity to track shipments by reference, package or shipment number. It is an easy way to get shipment details and events and make them available for customers. The information available in this API is the same that is publicly available from the [Tracking website](http://tracking.bring.com/).
+  The Tracking API provides the opportunity to track shipments by reference, package or shipment number. It is an easy way to get shipment details and events and make them available for customers. The information available in this API is the same that is publicly available from the [Tracking website](http://api.bring.com/).
```
Comment (richas2) [inline content/api/tracking/index.md:49]: Here we are talking about tracking website, not the API so it should be tracking.bring.com

File: content/api/tracking/index.md [hunk 2]
```diff
@@ -77,20 +77,19 @@ documentation:
...
       | Application | URL example |
       |:-------|:--------|
-      | Open tracking | https://tracking.bring.com/api/**v2**/tracking.json?q=TESTPACKAGEATPICKUPPOINT |
-      | Logged-in tracking | https://api.bring.com/tracking/api/**v2**/tracking.json?q=TESTPACKAGEATPICKUPPOINT |
+      | Tracking | https://api.bring.com/tracking/api/**v2**/tracking.json?q=TESTPACKAGEATPICKUPPOINT |
 
       #### Example request
       ```
-      curl -v https://tracking.bring.com/api/v2/tracking.json?q=TESTPACKAGEATPICKUPPOINT
+      curl -v https://api.bring.com/api/v2/tracking.json?q=TESTPACKAGEATPICKUPPOINT
```
Comment (richas2) [inline content/api/tracking/index.md:92]: Ohhh here we need to add /tracking also in the path, so the correct path would be
https://api.bring.com/tracking/api/v2/tracking.json?q=TESTPACKAGEATPICKUPPOINT,

File: content/api/tracking/index.md [hunk 3]
```diff
@@ -127,6 +126,6 @@ documentation:
       | tracking.xml | `application/xml;charset=utf-8` | `<ApiVersion>2</ApiVersion>` |
 
 
-oas: https://tracking.bring.com/api-docs/
+oas: https://api.bring.com/api-docs/
```
Comment (richas2) [inline content/api/tracking/index.md:129]: Is this even working? Shouldnt it be https://api.bring.com/tracking/api-docs/

**Comment by richas2** [comment]
Think build is failing because it couldn't successfully call swagger URL

**Pull request merged**